tutorials/lava/lib/peripherals/dvs/PropheseeCamera.ipynb

```diff
@@ -0,0 +1,313 @@
+     "output_type": "stream",
+     "text": [
+      "/usr/lib/python3/dist-packages/metavision_core_ml/preprocessing/viz.py:41: NumbaDeprecationWarning: \u001b[1mThe 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.\u001b[0m\n",
+      "  def _viz_events(events, img):\n"
+     ]
+    }
+   ],
+   "source": [
+    "import matplotlib\n",
+    "%matplotlib inline\n",
+    "\n",
+    "from lava.magma.core.run_configs import Loihi2SimCfg\n",
+    "from lava.magma.core.run_conditions import RunSteps\n",
+    "\n",
+    "from lava.lib.peripherals.dvs.prophesee import PropheseeCamera\n",
+    "from metavision_core.utils import get_sample\n",
+    "from metavision_sdk_cv import TrailFilterAlgorithm, ActivityNoiseFilterAlgorithm\n",
+    "from metavision_core.event_io import RawReader\n",
+    "from lava.lib.peripherals.dvs.transform import Compose, Downsample\n",
+    "\n",
+    "from utils import VisProcess"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1e456b29-9503-4546-adc7-37c8d511dcf3",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## Download raw event recording"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "602de9c0-4501-42b3-b604-589346b82b19",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "SEQUENCE_FILENAME_RAW = \"80_balls.raw\"\n",
+    "# if the file doesn't exist, it will be downloaded from Prophesee's public sample server\n",
+    "get_sample(SEQUENCE_FILENAME_RAW)\n",
+    "\n",
+    "reader = RawReader(SEQUENCE_FILENAME_RAW)\n",
+    "height, width = reader.get_size()\n",
+    "del reader"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1e72821b-418a-451d-a52d-1e33f96faccb",
+   "metadata": {},
+   "source": [
+    "## Basic usage\n",
+    "\n",
+    "Initialize PropheseeCamera using the path to a recording file if RAW format, in order to use a live camera, replace the path with an empty string (\"\")."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f13c3774-0132-4858-b10d-94c0bac212c0",
+   "metadata": {},
+   "source": [
+    "\n",
+    "<table>\n",
+    "<tr>\n",
+    "    <td> <img src=\"gifs/basic.gif\" alt=\"Drawing\" style=\"height: 250px;\"/> </td>\n",
+    "</tr>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "2ce2d8f2-a440-4d4f-a86b-b0b62d3fbee9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "# Init Processes\n",
+    "frame_input = PropheseeCamera(device=SEQUENCE_FILENAME_RAW,\n",
+    "                              sensor_shape=(height, width),\n",
+    "                              num_output_time_bins=1)\n",
+    "\n",
+    "recv = VisProcess(shape=frame_input.s_out.shape)\n",
```

Comment:
Please rename `VisProcess` (see above) and name the resulting variable the same, e.g.
```python
event_visualizer = EventVisualizer(...)
```